packages/scheduler-prisma/src/maintenance.ts

```diff
@@ -26,15 +26,16 @@ export default function initMaintenance(
 
     try {
       // Returns a grouped object of jobs that are pending (jobs that are not currently running and are due to run)
-      const pendingJobs = await client.$queryRaw<
-        { name: string; count: string }[]
-      >`
+      const pendingJobs =
```

Comment:
@sajidanower23 this applied on the sequelize side?

Reply:
It wasn't, but I just applied it there too